classes/PodsUI.php

```diff
@@ -3058,7 +3108,18 @@ public function manage( $reorder = false ) {
 									<?php esc_html_e( $filter_field['label'] ); ?>
 									</label>
 									<?php
-									echo PodsForm::field( 'filter_' . $filter, $value, 'pick', $options );
+									// Prevent p div issues.
+									echo str_replace(
```

Comment:
:no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'str_replace'.

<sub>hash: ceaa110293679c4373a00085c2fe1a4d</sub>